Samples/GettingStarted/BinaryClassification_SentimentAnalysis/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.ML;
+using Microsoft.ML.Models;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Trainers;
+using Microsoft.ML.Transforms;
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BinaryClassification_SentimentAnalysis
+{
+    internal static class Program
+    {
+        private static string AppPath => Path.GetDirectoryName(Environment.GetCommandLineArgs()[0]);
+        private static string TrainDataPath => Path.Combine(AppPath, @"..\..\..\..\datasets\", "imdb_labelled.txt");
+        private static string TestDataPath => Path.Combine(AppPath, @"..\..\..\..\datasets\", "yelp_labelled.txt");
+        private static string ModelPath => Path.Combine(AppPath, "Models", "SentimentModel.zip");
+
+        private static async Task Main(string[] args)
+        {
+            var model = await TrainAsync();
+
+            Evaluate(model);
+
+            var predictions = model.Predict(TestSentimentData.Sentiments);
+
+            var sentimentsAndPredictions =
+                TestSentimentData.Sentiments.Zip(predictions, (sentiment, prediction) => (sentiment, prediction));
+            foreach (var item in sentimentsAndPredictions)
+            {
+                Console.WriteLine(
+                    $"Sentiment: {item.sentiment.SentimentText} | Prediction: {(item.prediction.Sentiment ? "Positive" : "Negative")} sentiment");
+            }
+
+            Console.ReadLine();
+        }
+
+        public static async Task<PredictionModel<SentimentData, SentimentPrediction>> TrainAsync()
+        {
+            // LearningPipeline allows us to add steps in order to keep everything together 
+            // during the learning process.  
+            var pipeline = new LearningPipeline();
+
+            // The TextLoader loads a dataset with comments and corresponding postive or negative sentiment. 
+            // When you create a loader you specify the schema by passing a class to the loader containing
+            // all the column names and their types. This will be used to create the model, and train it. 
+            pipeline.Add(new TextLoader<SentimentData>(TrainDataPath, useHeader: false, separator: "tab"));
+
+            // TextFeaturizer is a transform that will be used to featurize an input column. 
+            // This is used to format and clean the data.
+            pipeline.Add(new TextFeaturizer("Features", "SentimentText"));
+
+            //add a FastTreeBinaryClassifier, the decision tree learner for this project, and 
+            //three hyperparameters to be used for tuning decision tree performance 
+            pipeline.Add(new FastTreeBinaryClassifier() {NumLeaves = 5, NumTrees = 5, MinDocumentsInLeafs = 2});
```

Comment:
I'd recommend Bigrams+Trichargrams w/ AveragedPerceptronBinaryClassifier{iter=10} for text. AveragedPerceptron generally wins on text vs. FastTree in terms of accuracy & speed.

Reply:
Thank you! I will use them for advance scenario samples. This one is a "Hello world" type of sample where I'm trying to make the pipeline as simple as possible. And in the next samples I'll demonstrate how the results can be improved with different transforms.